packages/main/src/RadioButton.js

```diff
@@ -167,6 +190,15 @@ class RadioButton extends UI5Element {
 	onBeforeRendering() {
 		this.syncLabel();
 		this.syncGroup();
+
+		if (RadioButton.FormSupport) {
```

Comment:
Seems great, do you mind extracting the form support code block in a separate method -enableFormSupport or something similar, just to keep the onBeforeRendering lean.

onBeforeRendering() {
		this.syncLabel();
		this.syncGroup();
         this.enableFormSupport();
}